src/code/z_play.c

```diff
@@ -110,34 +110,254 @@
 
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_play/func_80169C84.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play/convert_scene_number_among_shared_scenes.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play/func_80169D40.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play/func_80169DCC.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play/func_80169E6C.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play/func_80169ECC.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play/func_80169EFC.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play/func_80169F78.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play/func_80169FDC.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play/func_80169FFC.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play/FrameAdvance_IsEnabled.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play/func_8016A02C.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play/func_8016A0AC.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play/func_8016A168.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play/func_8016A178.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play/func_8016A268.s")
+/**
+ * Converts the number of a scene to its "original" equivalent, the default version of the area which the player first
+ * enters.
+ */
+s16 Play_GetOriginalSceneNumber(s16 sceneNum) {
+    // Inverted Stone Tower Temple  -> Stone Tower Temple
+    if (sceneNum == SCENE_INISIE_R) {
+        return SCENE_INISIE_N;
+    }
```

Comment:
Something that I've noticed for a while, the style that's most common throughout the repo is it use `id` as identification for an enum, not `num`. So shouldn't it be called `sceneId`?

Reply:
+1 here. Seems to make more sense for these to be sceneIds. Thought perhaps it would be better in a different PR.